json-definitions/v3/tech-record/enums/vehicleConfiguration.enum.json

```diff
@@ -0,0 +1,32 @@
+{
+    "title": "Vehicle Configuration",
+    "type": "string",
+    "tsEnumNames": [
+        "ARTICULATED",
+        "RIGID",
+        "CENTRE_AXLE_DRAWBAR",
+        "SEMI_CART_TRANSPORTER",
```

Comment:
spelling mistake here on SEMI_CART_TRANSPORTER
Once that is corrected, Craig is right, this is exactly the same as the Light Vehicle enum, so we don't need both